Dashboard/app/js/templates/navData/monitoring-data.handlebars

```diff
@@ -137,9 +137,87 @@
+                            var geoshapeObject = $("#geoshapeObject").data("geoshape");
+                            var geoshapeCoordinates = [], geoshapeMetadata = "";
+                            if(geoshapeObject !== "" && geoshapeObject !== null && geoshapeObject !== "null"){
+                              if(geoshapeObject['features'].length > 0){
+                                var geoshapeCoordinatesArray = geoshapeObject['features'][0]['geometry']['coordinates'][0];
+                                for(var j=0; j<geoshapeCoordinatesArray.length; j++){
+                                  geoshapeCoordinates.push([geoshapeCoordinatesArray[j][1], geoshapeCoordinatesArray[j][0]]);
+                                }
+                                createGeoshape(geoshapeCoordinates);
+
+                                geoshapeMetadata += '<div style="float: left; width: 100%">Points: '+geoshapeObject['features'][0]['properties']['pointCount']+'</div>';
+                                geoshapeMetadata += '<div style="float: left; width: 100%">Length: '+geoshapeObject['features'][0]['properties']['length']+'m</div>';
+                                geoshapeMetadata += '<div style="float: left; width: 100%">Area: '+geoshapeObject['features'][0]['properties']['area']+'m&sup2;</div>';
+                              }
+                            }
+                            $("#geoshapeMetadata").html(geoshapeMetadata);
+
+                            function createGeoshape(points){
+                              var getCentroid = function (arr) {
+                                return arr.reduce(function (x,y) {
+                                  return [x[0] + y[0]/arr.length, x[1] + y[1]/arr.length]
+                                }, [0,0])
+                              }
+
+                              var center = getCentroid(points), geoShape;
+
+                              // create leaflet map
+                              var geoshapeMap = L.map('geoShapeMap', {scrollWheelZoom: false}).setView(center, 2);
+
+                              geoshapeMap.options.maxZoom = 18;
+                              geoshapeMap.options.minZoom = 2;
+
+                              var mbAttr = 'Map &copy; 1987-2014 <a href="http://developer.here.com">HERE</a>',
+                          			mbUrl = 'https://{s}.{base}.maps.cit.api.here.com/maptile/2.1/maptile/{mapID}/{scheme}/{z}/{x}/{y}/256/{format}?app_id={app_id}&app_code={app_code}';
+
+                              var normal = L.tileLayer(mbUrl, {
+                                scheme: 'normal.day.transit',
+                                format: 'png8',
+                                attribution: mbAttr,
+                                subdomains: '1234',
+                                mapID: 'newest',
+                                app_id: FLOW.Env.hereMapsAppId,
+                                app_code: FLOW.Env.hereMapsAppCode,
```

Comment:
@mtwestra @janagombitova @kymni I wonder if we want to use heremaps or mapbox maps for the geoshapes here? Do we need to also support google maps (for the region bias problem)?

If we use here heremaps we need to add the `appId` and `appCode` to all the instances in `akvo-flow-server-config` since currently we only have these properties for instances that has Cartodb maps enabled.